configurations/create_kenya_pool.py

```diff
@@ -0,0 +1,219 @@
+                CodaDatasetConfiguration(
+                    coda_dataset_id="Kenya_Pool_location", #TODO Rename as Pool_Kenya_location
+                    engagement_db_dataset="location",
+                    code_scheme_configurations=[
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("demographics/kenya_constituency"), auto_coder=None),
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("demographics/kenya_county"), auto_coder=None)
+                    ],
+                    ws_code_string_value="location"
+                ),
+                CodaDatasetConfiguration(
+                    coda_dataset_id="Kenya_Pool_gender",
+                    engagement_db_dataset="gender",
+                    code_scheme_configurations=[
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("demographics/gender"),
+                                                auto_coder=swahili.DemographicCleaner.clean_gender)
+                    ],
+                    ws_code_string_value="gender"
+                ),
+                CodaDatasetConfiguration(
+                    coda_dataset_id="Kenya_Pool_age",
+                    engagement_db_dataset="age",
+                    code_scheme_configurations=[
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("demographics/age"), auto_coder=lambda x:
+                                                str(swahili.DemographicCleaner.clean_age_within_range(x)))
+                    ],
+                    ws_code_string_value="age"
+                ),
+                CodaDatasetConfiguration(
+                    coda_dataset_id="Kenya_Pool_disabled",
+                    engagement_db_dataset="disabled",
+                    code_scheme_configurations=[
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("demographics/disabled"), auto_coder=None)
+                    ],
+                    ws_code_string_value="disabled"
+                ),
+                CodaDatasetConfiguration(
+                    coda_dataset_id="Kenya_Pool_old_rqa_datasets",
+                    engagement_db_dataset="kenya_pool_old_rqa_datasets",
+                    code_scheme_configurations=[
+                        CodeSchemeConfiguration(code_scheme=load_code_scheme("kenya_pool_old_rqa_datasets"), auto_coder=None)
+                    ],
+                    ws_code_string_value="kenya_pool_old_rqa_datasets"
+                ),
+            ],
+            ws_correct_dataset_code_scheme=load_code_scheme("ws_correct_dataset"),
+            project_users_file_url="gs://avf-project-datasets/2022/POOL-KENYA/pool-kenya-users.json",
+            default_ws_dataset="kenya_pool_old_rqa_datasets"
+        )
+    ),
+    archive_configuration=ArchiveConfiguration(
+        archive_upload_bucket="gs://pipeline-execution-backup-archive",
+        bucket_dir_path="2022/CREATE-KENYA-POOL/"
```

Comment:
```suggestion
        bucket_dir_path="2022/CREATE-KENYA-POOL"
```
otherwise files will have a // in their path in the bucket, which makes listing them a bit harder.